cleverhans/attacks.py

```diff
@@ -412,6 +412,90 @@ def parse_params(self, theta=1., gamma=np.inf, nb_classes=10, clip_min=0.,
         return True
 
 
+class VirtualAdversarialMethod(Attack):
```

Comment:
Can you add at least a test that makes sure the class can be instantiated without crashing?